src/main/java/core/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,53 @@
+package core.controller.car;
+
+import core.lib.Injector;
+import core.model.Car;
+import core.model.Driver;
+import core.service.car.CarService;
+import core.service.driver.DriverService;
+import java.io.IOException;
+import java.util.NoSuchElementException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector
+            = Injector.getInstance("core");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+    
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/add_driver_to_car.jsp").forward(req, resp);
+    }
+    
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String carId = req.getParameter("carId");
+        String driverId = req.getParameter("driverId");
+        if (carId.isEmpty() || driverId.isEmpty()) {
+            errorInput(req, resp, "Empty input provided");
```

Comment:
```suggestion
            errorInput(req, resp, "Empty input provided");
            return;
```
we should not be able to go further in this method

Reply:
I thought of that redirect in errorInput() method breaks the further execution of the method. Thanks, good to know